addons/website_event_track/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,51 @@
+---Fields in module 'website_event_track'---
+website_event_track / event.track              / activity_ids (one2many)       : NEW relation: mail.activity
+website_event_track / event.track              / kanban_state (selection)      : NEW required: required, selection_keys: ['blocked', 'done', 'normal'], req_default: function
+# Nothing to do
+
+website_event_track / event.track              / speaker_ids (many2many)       : DEL relation: res.partner
+# Nothing to do
```

Comment:
if the event track record doesn't have a partner_id, then you should fill it with one of the speaker_ids